### PR TITLE
Created the endpoint to GET a user by ID

### PR DESCRIPTION
**Pull request by AnaBennett11**
### What (if any) features are you implementing?
-Created the endpoint to GET a user by ID to get the user's ID, email and password

### Refactor checklist
- [x] Check variable and parameter names: are they descriptive and unique?
- [x] Is the code DRY? SRP?
- [x] Is it easy for others to read your code? (explanatory comments needed?)
- [x] Is the syntax consistent? (e.g. use of semicolons, blank lines)
- [x] Are commit messages formatted correctly?
Screenshot 2023-02-28 at 4 56 03 PM

### CODE REVIEW

**Review by susannaopal — APPROVED** [review]
Thanks for working on this!

File: backend/server.js [hunk 1]
```diff
@@ -20,6 +20,18 @@ app.get("/", (req, res) => {
...
 });
 
+app.get("/:userID", (req, res) => {
+  const userID = req.params.userID;
+  const userData = JSON.parse(fs.readFileSync(userDB));
+  const user = userData.find((user) => user.userId === userID);
+  const { userId, userEmail, userPassword } = user;
+  if (!user) {
+    res.status(404).send("User not found");
+  } else {
```
Comment (YashsviG) [inline backend/server.js:30]: This part should come before line 27, this might actually throw an error if the user doesnt exist

Comment (YashsviG) [reply backend/server.js:30]: Also, can you please a try-catch block just in case there are un-foreseen errors they can all be handled appropriately

**Review by YashsviG — CHANGES_REQUESTED** [review]
Can you also please attach a postman screenshot to show this works as expected

File: backend/server.js (hunk 1 above)
Comment (AnaBennett11) [reply backend/server.js]: when a user is not found:
Screenshot 2023-03-01 at 3 12 16 PM
when a user is found:
Screenshot 2023-03-01 at 3 12 31 PM

Comment (AnaBennett11) [reply backend/server.js]: Screenshot 2023-03-01 at 3 14 13 PM

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,4894 @@
+{
```
[the remark this answers is not in the document]
Comment (AnaBennett11) [reply package-lock.json]: Thanks so much! Just removed it and pushed my changes!

**Review by YashsviG — APPROVED** [review]
Great work!

**Pull request merged**